.github/workflows/ci.yml

```diff
@@ -50,6 +50,8 @@ jobs:
       # be incompatible with some of these targets.
       - name: Create synthetic crate for testing
         run: cargo init --lib ci-build
+      - name: Set edition for compatibility
+        run: sed -i 's/edition = .*/edition = "2021"/' ci-build/Cargo.toml
```

Comment:
should we remove it later?
how come it works for upstream but not for us?

Reply:
Since their last commit github's runners have been upadted to use Rust 1.85/2024 and here we try to create older artifact that cannot owrk with edition 2024. We probably can bump up Rust version in the project to solve that